src/database/migrations/20230912185913_AddDeletedFlagToUsersTable.ts

```diff
@@ -0,0 +1,16 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.alterTable('users', (table) => {
+    table.boolean('deleted').defaultTo(false).notNullable();
+  });
+
+  // Update existing rows to have the default value for 'completedRecommendations'
```

Comment:
nit: update comment :)

Reply:
I fixed the comment and filtered deleted users where necessary.